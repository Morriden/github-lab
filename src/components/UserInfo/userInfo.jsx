import React from 'react';
import PropTypes from 'prop-types';

const userInfo = ({ userName, followers, following, link }) => (
  <section>
    <div>{userName}</div>
    <div>{followers}</div>
    <div>{following}</div>
    <div>{link}</div>
  </section>
);

userInfo.propTypes = {
  userName: PropTypes.string.isRequired,
  followers: PropTypes.number.isRequired,
  following: PropTypes.number.isRequired,
  link: PropTypes.string.isRequired
};

export default userInfo;
