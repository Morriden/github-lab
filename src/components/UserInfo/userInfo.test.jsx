import React from 'react';
import { shallow } from 'enzyme';
import userInfo from './userInfo';

describe('Form component', () => {
  it('renders App', () => {
    const wrapper = shallow(<userInfo userName="Morriden" followers={1} following={2} link="url string" />);
    expect(wrapper).toMatchSnapshot();
  });
});
