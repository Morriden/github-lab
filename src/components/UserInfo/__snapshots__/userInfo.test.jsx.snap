// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component renders App 1`] = `
<userInfo
  followers={1}
  following={2}
  link="url string"
  userName="Morriden"
/>
`;
